packages/logo/src/Logos/AtlasLogoMark.tsx

```diff
@@ -1,170 +1,93 @@
 import React from 'react';
 
 import { css, cx } from '@leafygreen-ui/emotion';
+import { palette } from '@leafygreen-ui/palette';
 
-import { ProductLogoProps } from '../Logo.types';
-import { getAccessibleProps, getColor } from '../utils';
+import { ProductLogoProps, SupportedColorsMap } from '../Logo.types';
+import { getAccessibleProps } from '../utils';
 
-/**
- * @deprecated
- */
 export const AtlasLogoMark = React.forwardRef(
   (
     {
-      size = 18,
-      knockout = false,
+      height = 33,
       darkMode = false,
       role = 'img',
       'aria-label': ariaLabel = 'Atlas LogoMark',
       className,
+      color,
       ...rest
     }: ProductLogoProps,
     ref: React.LegacyRef<SVGSVGElement> | undefined,
   ) => {
+    const fill = color
+      ? SupportedColorsMap[color]
+      : darkMode
+      ? palette.green.base
+      : palette.green.dark2;
+
     return (
       <svg
         {...getAccessibleProps({ role, 'aria-label': ariaLabel })}
-        x="0px"
-        y="0px"
-        viewBox="0 0 18 18"
-        width={size}
-        height={size}
+        height={height}
```

Comment:
I was wrong! This should match the `viewBox` hard-coded numbers (otherwise the logo will warp!)